common/menu-entry.c

```diff
@@ -293,6 +294,13 @@ bool menuEntryLoad(menuEntry_s* me, const char* name, bool shortcut) {
         /*if (shortcut)
             shortcutFree(&sc);*/
     }
+    if (me->type == ENTRY_TYPE_THEME){
+        
+        //Todo load in the data about the theme?
+        //Add a theme name property to each one?
+        //Ability to load the theme as you hover over?
+        free((void*)name);//This was allocated by calloc and copied over by strcpy
```

Comment:
Don't free that here, do it in the caller.

Reply:
so if(menu_entry->type == Theme) free(name)? in the menuEntryFree function?

Reply:
That really doesn't need to be on the heap, should be a stack buffer in themeMenuScan.